_rules/device-motion-disabled-c249d5.md

```diff
@@ -28,24 +28,19 @@ htmlHintIgnore:
 
 The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
 
-## Expectation 1
+## Expectation
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] is available to disable the event.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] to prevent the outcome of the event is available in the same [web page][], or in another [web page][] that is [linked][hyperlink] from the [web page][] of the test target.
```

Comment:
Actually, the SC only says that the functionality "can be disabled". It doesn't say anything about instrument (UI component). (while the "alternate operation" part does refer to a UI component).

I think we are making an assumption that "can be disabled" means "by a UI component".
A strict reading of the SC, providing a phone number to a support team that will server-side disable the motion detection passes the SC… (but not this rule).

And if the rule assumes that "can be disabled" means "by a UI component", then I think that expectations 2 and 3 should stay (as UI component "can be perceived as …" which is pretty difficult for something with `display: none`).

Reply:
Agree. Will add an assumption and reinstate expectations 2 and 3.